erizo_controller/erizoJS/models/Subscriber.js

```diff
@@ -51,9 +51,14 @@ class Subscriber extends NodeClass {
         Math.min(this.publisher.getMaxVideoBW(), this.maxVideoBW) : this.maxVideoBW;
       this.maxVideoBW = updatedMaxVideoBW;
     } else {
-      updatedMaxVideoBW = this.publisher.getMaxVideoBW();
+      // eslint-disable-next-line no-lonely-if
+      if (this.publisher.getMaxVideoBW) {
+        updatedMaxVideoBW = this.publisher.getMaxVideoBW();
+      } else {
+        updatedMaxVideoBW = 0;
```

Comment:
This means all subscribers will have a maxVideoBW of 0 for this stream. Have you checked that this causes no issues? AFAIK we don't really support not having a value here. It does have less implications when not in simulcast like in this case, but it's weird nonetheless. We would be generating REMBs with 0.
I would implement a solution where we set a maxVideoBW for the ExternalInput.